File: refactored/assertion-roulette/python-pytest/TestEventHandlerService_testService.py
import pytest

class TestEventHandlerService:
    def test_service(self):
        ehs = _test_event_handler_service()
        assert isinstance(ehs.get_event_queue(), MemoryEventQueue), "Explanation message"
        jobtypes = ehs.get_app_types()
        assert "workflow_job" in jobtypes, "Explanation message"
        assert "coordinator_action" in jobtypes, "Explanation message"

        services = Services.get()
        services.destroy()
        services = Services()
        conf = services.get_conf()
        conf.set(Services.CONF_SERVICE_EXT_CLASSES, "")
        services.init()
        assert not EventHandlerService.is_enabled(), "Explanation message"

File: refactored/assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/TestEventHandlerService_testService.cs
public class TestEventHandlerService : XDataTestCase
{
    [Fact]
    public void TestService()
    {
        EventHandlerService ehs = _testEventHandlerService();
        // check default initializations
        Assert.True(ehs.GetEventQueue() is MemoryEventQueue, "Explanation message");
        HashSet<string> jobtypes = ehs.GetAppTypes();
        Assert.True(jobtypes.Contains("workflow_job"), "Explanation message");
        Assert.True(jobtypes.Contains("coordinator_action"), "Explanation message");

        Services services = Services.Get();
        services.Destroy();
        services = new Services();
        Configuration conf = services.GetConf();
        conf.Set(Services.CONF_SERVICE_EXT_CLASSES, "");
        services.Init();
        Assert.False(EventHandlerService.IsEnabled(), "Explanation message");
    }
}
